### PR TITLE
add measure identifier to bundle identifier

### PR DESCRIPTION
**Pull request by mdnazmulkarim**
**Description**

- Github Issue:  
- [x] I've read the contribution guidelines 
- [x] Code compiles without errors
- [ ] Tests are created / updated
- [ ] Documentation is created / updated

By creating this PR you acknowledge that your contribution will be licensed under Apache 2.0

### CODE REVIEW

File: src/main/java/org/opencds/cqf/tooling/measure/r4/R4MeasureProcessor.java [hunk 1]
```diff
@@ -64,6 +64,9 @@ else if (file.isDirectory()) {
         List<org.hl7.fhir.r5.model.Measure> refreshedMeasures = super.refreshGeneratedContent(measures);
         VersionConvertor_40_50 versionConvertor_40_50 = new VersionConvertor_40_50(new BaseAdvisor_40_50());
         for (org.hl7.fhir.r5.model.Measure refreshedMeasure : refreshedMeasures) {
+            if(refreshedMeasure.hasIdentifier()) {
```
Comment (JPercival) [inline src/main/java/org/opencds/cqf/tooling/measure/r4/R4MeasureProcessor.java:67]: We probably want to surface all identifiers

File: src/main/java/org/opencds/cqf/tooling/measure/MeasureProcessor.java [hunk 2]
```diff
@@ -62,6 +63,10 @@ public List<String> refreshIgMeasureContent(BaseProcessor parentContext, Encodin
         return measureProcessor.refreshMeasureContent(params);
     }
 
+    protected Object getIdentifier() {
```
Comment (JPercival) [inline src/main/java/org/opencds/cqf/tooling/measure/MeasureProcessor.java:66]: Let's make this a List of whatever dataType the Identifiers are.

File: src/main/java/org/opencds/cqf/tooling/measure/r4/R4MeasureProcessor.java [hunk 3]
```diff
@@ -64,6 +64,9 @@ else if (file.isDirectory()) {
...
+                this.identifier = refreshedMeasure.getIdentifier().get(0);
```
Comment (JPercival) [inline src/main/java/org/opencds/cqf/tooling/measure/r4/R4MeasureProcessor.java:68]: Also, including the identifier should be optional

**Review by JPercival — CHANGES_REQUESTED** [review]

**Comment by brynrhodes** [comment]
Getting warnings on this trying to build locally, which fails the build at this point... please address the warnings

**Comment by brynrhodes** [comment]
I also don't think it should be _just_ the measure identifier, can we append -bundle to the identifier to make it clear that it's the identifier for the bundle, not the measure?

**Pull request merged**